### PR TITLE
More complete support for Livolo cover

### PR DESCRIPTION
**Pull request by aesedepece**
Thanks to https://github.com/Koenkk/zigbee-herdsman/pull/33, this adds support for:
- STOP command
- adjusting motor speed
- adjusting motor direction
- exposing motor movement info

This is what the options look like now:
image

### CODE REVIEW

**Comment by aesedepece** [comment]
The linter complains about this PR but the offending line actually comes from the master branch. See #2408 :stuck_out_tongue_winking_eye:

**Comment by aesedepece** [comment]
Added one more commit that improves handling of motor direction and initialization of some settings. I found these changes were necessary when playing with a second brand new unit of this device.

**Comment by Koenkk** [comment]
Thanks!

File: converters/toZigbee.js [hunk 1]
```diff
@@ -1457,37 +1457,90 @@ const converters = {
...
+            case 'CLOSE':
+                payload =
+                    {attrId: 0x0000, selector: null, elementData: [0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]};
+                break;
+            case 'STOP':
+                payload =
+                    {attrId: 0x0000, selector: null, elementData: [0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]};
+                break;
+            default:
+                throw new Error(`Value '${value}' is not a valid cover position (must be one of 'OPEN' or 'CLOSE')`);
```
Comment (sjorge) [inline converters/toZigbee.js:1481]: Should this also include 'STOP' as that seems to be a valid value as there is a case and payload for it above. (Although yes, STOP would not be a valid 'position' it is still a valid value)

Comment (aesedepece) [reply converters/toZigbee.js:1481]: That's true

**Comment by lotohov** [comment]
Hello , tried to test new integration , but my covers detected as switch Zigbee2MQTT:info  2021-05-09 23:25:18: Device '0x00124b0021753df6' is supported, identified as: Livolo Zigbee switch (1 and 2 gang) (TI0001), how i can fix it ? I have 1.18.3 version

**Comment by aesedepece** [comment]
Simply click "reconfigure" from the Zigbee2mqtt UI. Sometimes it takes a
couple of attempts, but it should eventually detect the device as a cover
instead of a switch.

**Pull request merged**